pyqtgraph/functions.py

```diff
@@ -2320,7 +2358,8 @@ def invertQTransform(tr):
     try:
         import numpy.linalg
```

Comment:
Import can be omitted in this implementation

Reply:
Actually, if the import is omitted, the entire try/except can bee omitted...

Reply:
Sweet! Will do.

Reply:
@ksunden i'm seeing the `tr.inverted()` use in the `except:`  and [reading the docs for it](https://doc.qt.io/qt-5/qtransform.html#inverted) I'm wondering do we even need our own inversion at all?

Gonna put up benchmarks against both.